pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunction.java

```diff
@@ -228,7 +229,8 @@ public TransformResultMetadata getResultMetadata() {
     return _resultMetadata;
   }
 
-  private boolean isNullLiteralTransformation(TransformFunction function) {
+  @VisibleForTesting
```

Comment:
We want to test private methods via public APIs.

https://abseil.io/resources/swe-book/html/ch12.html, section "Test via Public APIs".